### DISCUSSION TITLE
MQTT Setup

### DISCUSSION THREAD

**Discussion by gary-hammer** [Daveiano/weewx-wdc#146] [Q&A] [answered]
I am excited to get live data in this skin.
While there are problems with the skin in my install, is my stanza correct as to indentation?
See attached file wdc-example.txt

Thanks for the live data!

[wdc-example.txt](https://github.com/Daveiano/weewx-wdc/files/11129952/wdc-example.txt)

**Comment by Daveiano** [comment, marked as answer]
No, It should be:

```
[[WdcReport]]
    skin = weewx-wdc
    enable = true
    lang = en
    HTML_ROOT = /home/weather/wdc
    [[[Extras]]]
        [[[[mqtt]]]]
            mqtt_websockets_enabled = 1
            mqtt_websockets_host = wx-tab.lan
            mqtt_websockets_port = 9001
            mqtt_websockets_ssl = 0
            mqtt_websockets_topic = weather/loop
```

**Comment by gary-hammer** [comment]
Thank you, all is working well now.
Looks and runs great!

Now to look at radar and forecast.

**Reply by Daveiano** [reply]
Glad it is working now! Happy to help!
